AdyenCard/Form/FormCardNumberItem.swift

```diff
@@ -46,7 +46,14 @@ internal final class FormCardNumberItem: FormTextItem, AdyenObserver {
     internal init(cardTypeLogos: [FormCardLogosItem.CardTypeLogo],
                   style: FormTextItemStyle = FormTextItemStyle(),
                   localizationParameters: LocalizationParameters? = nil) {
-        self.cardTypeLogos = cardTypeLogos
+        // these 4 US debit brands are not to be displayed
+        // but should be supported so it's done here for now
+        self.cardTypeLogos = cardTypeLogos.filter { logo in
+            logo.type != .accel &&
+                logo.type != .pulse &&
+                logo.type != .star &&
+                logo.type != .nyce
+        }
```

Comment:
```suggestion
        self.cardTypeLogos = cardTypeLogos.filter { [.accel, .pulse, .star, .nyce].contains($0.type) == false }
```

Reply:
Nice suggestion! But it requires to conform to `Hashable`. Think we should add just for this?

Reply:
I don't think Hashable is needed, I was able to do the same change on develop and it builds

Reply:
Oh I replied too fast because your original suggestion was to use `minus` so I assumed this is the same without reading :D